airflow/models/dag.py

```diff
@@ -1850,31 +1562,27 @@ def create_dagrun(self,
     def set_is_paused(self,
                       is_paused: bool,
                       including_subdags: bool = True,
-                      store_serialized_dags: bool = False,
                       session=None) -> None:
         """
         Pause/Un-pause a DAG.
 
         :param is_paused: Is the DAG paused
         :param including_subdags: whether to include the DAG's subdags
-        :param store_serialized_dags: whether to serialize DAGs & store it in DB
         :param session: session
         """
-        dag_ids = [self.dag_id]  # type: List[str]
+        filter_query = [
+            DagModel.dag_id.in_(self.dag_id),
+        ]
         if including_subdags:
-            dag = self.get_dag(store_serialized_dags)
```

Comment:
I'm not sure if this behavior is 100% identical, but it seems sensible from a web UIi perspective.